resources/devnet/main.tf

```diff
@@ -37,6 +37,23 @@ module "devnet" {
     disk-volume-type   = var.disk_volume_type
   }
 
+  bootstrap-node-autoid-config = {
```

Comment:
I was actually hoping we can merge the nova and autoID in to a single type. There is not much difference except domain-id

Reply:
If we are running both concurrently how will it work?

> This is not correct unfotunately

We would need to create two compose file. One for EVM and one for AutoID
Our node currently does not support running multiple domains in the single binary